test/images/scripts/centos-7.bootstrap

```diff
@@ -1,4 +1,7 @@
 #! /bin/bash
 
 BASE=$(dirname $0)
-$BASE/virt-builder-fedora "$1" centos-7.3 x86_64
+# get the latest centos-7 virt-builder has to offer
+CENTOS_LATEST=$(virt-builder -l | sort -r | grep -m 1 centos-7 | cut -d" " -f1)
```

Comment:
I think you should extend the grep for `centos-7.*x86_64` to avoid catching e. g. the `aarch64` or any other architecture image, just in case that gets available earlier.